reference/replica-operations.md

```diff
@@ -2,27 +2,26 @@
+Once a volume is processing I/O, each of its replicas will also receive I/O.  Reads are round-robin distributed across replicas, whilst writes must be written to all.  In a real world environment this is attended by the possibility that I/O to one or more replicas might fail at any time.  Possible reasons include transient loss of network connectivity, node reboots, node or disk failure.  If a volume's nexus \(NVMe controller\) encounters 'too many' failed I/Os for a replica, then that replica's child status will be marked `Faulted` and it will no longer receive I/O requests from the nexus.  It will remain a member of the volume, whose departure from the desired state with respect to replica count will reflected with a volume status of `Degraded`.  How many I/O failures are considered "too many" in this context is outside the scope of this discussion.
 
-The control plane will attempt to restore the desired state \(replica count\) by creating a new replica, following its replica placement rules.  If it succeeds, the nexus will "rebuild" that new replica - performing a full copy of all data from a healthy replica `CHILD_ONLINE`, i.e. the source.  This process can proceed whilst the volume continues to process application I/Os although it will contend for disk throughput at both the source and destination disks.  It will also 'retire' the old, faulted one which will then no longer be associated to the volume.  Once retired, a replica will become available for garbage collection (deletion from the Mayastor Pool containing it), assuming that the nature of the failure was such that the pool itself is still viable (i.e. the underlying disk device is still accessible).  N.B.: For a replica to be retired from a volume, it must have been possible to create a suitable replacement first.  In the absence of a replacement, the replica remains a member of the nexus, albeit faulted and thus unusable.
+The control plane will first 'retire' the old, faulted one which will then no longer be associated to the volume. Once retired, a replica will become available for garbage collection (deletion from the Mayastor Pool containing it), assuming that the nature of the failure was such that the pool itself is still viable (i.e. the underlying disk device is still accessible).  N.B.: For a replica to be retired from a volume, it must have been possible to create a suitable replacement first.  In the absence of a replacement, the replica remains a member of the nexus, albeit faulted and thus unusable.
```

Comment:
I think we need to remove the `N.B.` section as we can now retire a replica before finding a suitable replacement.